docs/detections/prebuilt-rules/downloadable-packages/0-16-2/prebuilt-rule-0-16-2-linux-restricted-shell-breakout-via-the-vi-command.asciidoc

```diff
@@ -0,0 +1,66 @@
+[[prebuilt-rule-0-16-2-linux-restricted-shell-breakout-via-the-vi-command]]
+=== Linux Restricted Shell Breakout via the vi command
+
+Identifies Linux binary find abuse to break out from restricted environments by spawning an interactive system shell. The vi/vim is the standard text editor in Linux distribution and the activity of spawning a shell is not a standard use of this binary by a user or system administrator and could potentially indicate malicious actor attempting to improve the capabilities or stability of their access."
```

Comment:
```suggestion
Identifies Linux binary abuse to break out from restricted environments by spawning an interactive system shell. The vi/vim is the standard text editor in Linux distribution. The activity of spawning a shell is not a standard use of this binary by a user or system administrator and could potentially indicate malicious actor attempting to improve the capabilities or stability of their access.
```

Reply:
Wasn't sure whether "find" was leftover text, so please ignore my edit and keep it if it's not.